pyaedt/icepak.py

```diff
@@ -1504,20 +1514,19 @@ def create_assign_region2PCBComponent(self, scale_factor = 1.0, restore_padding_
 
     @aedt_exception_handler
     def create_temp_point_monitor(self, point_name, point_coord = [0,0,0]):
-        """Create a temperature monitor for simulation.
+        """Create a temperature monitor for the simulation.
 
         Parameters
         ----------
         point_name : str
             Name of the temperature monitor.
         point_coord : list, optional
-            Coordinate of the point monitor. The default is ``[0,0,0]``.
+            Point coordinates for the temperature monitor. The default is ``[0,0,0]``.
```

Comment:
```suggestion
            Point coordinates for the temperature monitor. The default is ``[0, 0, 0]``.
```
Remember to separate items in a list by adding a single space after each comma.